frontend/model/contracts/chatroom.js

```diff
@@ -301,6 +301,18 @@ sbp('chelonia/defineContract', {
         //       a potential deadlock. See same warning in sideEffect for
         //       'gi.contracts/group/removeMember'
         await leaveChatRoom(contractID, state)
+        const me = sbp('state/vuex/state').loggedIn.identityContractID
+        if (me === state.attributes.creatorID || state.attributes.adminIDs.includes(me)) {
```

Comment:
Are you sure we should be checking `state.attributes.adminIDs`?

Just because someone's an admin, doesn't mean they're paying for the contract, right? So `billableContractID` might not apply in that case

Reply:
Agreed, but I don't see many options. What do you suggest we do? We have no way of knowing who the owner is, as it's not in the contract. The way it's written assumes that the owner will be either the creator or admin, so that they can issue the deletion command. I added the check to reduce the number of calls that will likely fail.

Reply:
If it's safe to do this then multiple times (and by running those code more frequently you're trying to guarantee at least someone who is the owner will successfully execute this code), then at the very least a comment is needed here explaining this situation. Explaining why some of the calls will fail and why that's expected and being done this way.

Reply:
Sure. Should I remove the check and let 'anyone' attempt to delete it? Or leave it as-is, assuming that the owner will at least be an admin or creator?